docs/configuration.md

```diff
@@ -53,6 +53,7 @@ spec:
     type: etcd
     etcd:
       peerAddress: 192.168.68.104
+      extraArgs: []
```

Comment:
```suggestion
      extraArgs: {}
```
After all, it's an `map[string]string`, right?

Reply:
Ah yes. every other extraArgs was also wrong, so I'm fixing the rest in a separate commit.